addons/mail/static/src/models/user_setting/user_setting.js

```diff
@@ -288,5 +312,8 @@ registerModel({
             inverse: 'userSetting',
             isCausal: true,
         }),
+        volumeSettingsTimeouts: attr({
+            default: {},
+        }),
```

Comment:
still have object in field even after the other split

Reply:
This shows we need to move this timeout into the record of each target participant.

Reply:
Yes it should, but I much prefer keeping PR small and to the point: turn prop into field.

In following PR, could be turned into field on `Partner`. Also change signature of some `Rtc` methods so it uses `Partner` rather than `Partner/id` as param (e.g. `Rtc/saveVolumeSetting`).